builders/integrations/bridges/axelar.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Going Cross-Chain Via Axelar Network
+description: Learn how to bridge assets, plus other ways you can connect your Moonbeam dApp to assets and functions on multiple blockchains using Axelar network & APIs.
+---
+
+# Axelar Network
+
+## Introduction 
+
+Axelar delivers secure cross-chain communication for Web3. Our infrastructure enables dApp users to interact with any asset or application, on any connected chain, with one click. Powered by a permissionless Proof-of-Stake validator set, Axelar network allows dApps to pass arbitrary messages across chains using Turing-complete calls.
+
+Axelar is a full-stack transport layer that supports asset transfer, General Message Passing and composability of programs. It securely connects all blockchain ecosystems, applications, assets and users to deliver Web3 interoperability. Axelar is composed of a decentralized network of validators, secure gateway contracts, uniform translation, routing architecture, and a suite of protocols and application programming interfaces (APIs). Take a look at the [Tech Stack Diagram](https://axelar.network/blog/an-introduction-to-the-axelar-network) for more details.
```

Comment:
Any external links should have {target=_blank} chained on to them so it opens the tab in a new window (i.e. `[Tech Stack Diagram](https://axelar.network/blog/an-introduction-to-the-axelar-network){target=_blank}`

Reply:
Thanks for pointing this out, will change!